src/_pytest/logging.py

```diff
@@ -431,13 +436,48 @@ def clear(self) -> None:
         """Reset the list of log records and the captured log text."""
         self.handler.reset()
 
+    def force_enable_logging(
+        self, level: Union[int, str], logger_obj: logging.Logger
+    ) -> int:
+        """Enable the desired logging level if the level was disabled.
+
+        Only enables logging levels equal too and higher than the requested ``level``.
```

Comment:
```suggestion
        Only enables logging levels equal to and higher than the requested ``level``.
```

Reply:
Well that's embarrassing. I'll fix it along with all other typos.

Reply:
fixed